ColdCardWallet.md

```diff
@@ -0,0 +1,43 @@
+# Connecting Coldcard Wallet to BTCPay Server
+
+This documents shows how to connect Coldcard Wallet to BTCPay Server. 
+
+## ColdCard Wallet Setup
+
+This guide assumes, you have a Coldcard wallet set up. To configure the Coldcard, please see the [quick setup guide on manufacturer's website](https://coldcardwallet.com/docs/quick).
+
+### Quick Setup
+
+1. Insert the MicroSD card into the Coldcard wallet.
+2. Go to Advanced > MiroSD Card > Electrum Wallet > Native Segwit
+3. Insert the MicroSD card back into your PC
+4. In BTCPay Server, Store Settings > General > Derivation Scheme > Import from Hardware Device > Coldcard
+5. Browse and select the wallet file earlier exported from Coldcard. 
+6. Click `Submit`
+7. Confirm the address match in BTCPay.
+
+The Coldcard is now connected to your BTCPay. Payments go directly to Coldcard. The video below shows how to connect your BTCPay store to your Coldcard.
+
+[![BTCPay and Coldcard](https://img.youtube.com/vi/N0eVwdP_7EQ/mqdefault.jpg)](https://www.youtube.com/watch?v=N0eVwdP_7EQ "Connecting Coldcard to BTCPay")
+
+###  Spending from BTCPay Server wallet with Coldcard (PSBT)
+
+Once there are some funds in received to your BTCPay Wallet connected to Coldcard, you can spend them by using [PSBT](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md#psbt-in-general) (Partially Signed Bitcoin Transactions). This allows completely offline signing of your transactions, without ever having to connect your hardware wallet to the internet.
+
+1. Wallets > Manage > Send
+2. Fill in destination address and the amount
+3. Click on the Sign with `a wallet supporting PSBT` button.
+4. You will be redirected to PSBT tab, with pre-filled information, click on the  Sign with `a wallet supporting PSBT (save as file)`
```

Comment:
(the  Sign) extra space
Also did you mean to include `Sign with` in the code formatted text line